src/zenml/integrations/kaniko/service_connectors/kaniko_service_connectors.py

```diff
@@ -0,0 +1,65 @@
+import base64
```

Comment:
This kaniko service connector is completely irrelevant:

* it's incomplete
* you're not really using it
* you don't even need it because what you really need is a Kubernetes service connector, which you're already using